.gitignore

```diff
@@ -1,3 +1,6 @@
 /vendor/
 /build
 composer.lock
+
+# IDE-specific
```

Comment:
Why is this change required? Please, add environment specific files to your [global ignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) file instead.